README.md

```diff
@@ -91,13 +91,15 @@ jobs:
     EOF
     ```
   
-1. Autofix code formatting errors using those linters with a built-in `fix` option. Note, this will create a `dirty` git directory,  
-   and you will need to manage committing and pushing any changes; consider setting `INPUT_AUTO_FIX=true` locally instead.
+1. Autofix code formatting errors using those linters with a built-in `fix` option. Note, this is enabled by default  
+   when running locally and will create a `dirty` git directory. You will need to manage committing and pushing any  
+   changes. Setting `INPUT_AUTO_FIX=false` locally will disable autofix and setting the action's auto_fix option to  
```

Comment:
No need for the extra explainer in README.md, it is covered in CONTRIBUTING.  Normal use case doesn't care about running locally. 
```suggestion
   changes.
```